xmtp/src/utils.rs

```diff
@@ -21,3 +22,7 @@ pub fn build_envelope(content_topic: String, message: Vec<u8>) -> Envelope {
         timestamp_ns: get_current_time_ns(),
     }
 }
+
+pub fn base64_encode(bytes: &[u8]) -> String {
+    general_purpose::STANDARD_NO_PAD.encode(bytes)
```

Comment:
Is there a reason to default to no padding?

Reply:
Did not give it a ton of thought. I just prefer NO_PAD aesthetically